content/api.md

```diff
@@ -4,15 +4,150 @@ title: API usage
 ---
 ## Introduction
 
-The data collected for the [2FA Directory](https://2fa.directory) website is also available as JSON files to enable developers to use it in their programs. The API with the highest version number is recommended since older versions might not include all available information.
+The data collected for the [2FA Directory][site_url] website is also available as JSON files for developers to use in their programs.
 
-### Caching
+## Supported versions
 
-If you intend to query our JSON files often and with a lot of traffic, you may be blocked by Cloudflare, our reverse proxy provider. We therefore recommend that you cache the files locally for any significant traffic cases.
+The table below provides the End of Life (EOL) dates for different 2FA Directory API versions. Please refer to this table to ensure you are using a supported version.
 
-### Avoid downloading unnecessary data
+|     Version      | Active Support | End of Life Date |
+|:----------------:|:--------------:|:----------------:|
+| [Version 4](#v4) |       ✅        |       N/A        |
+| [Version 3](#v3) |       ✅        |       N/A        |
+| [Version 2](#v2) |       ❌        |    2023-08-01    |
+| [Version 1](#v1) |       ❌        |    2023-07-01    |
```

Comment:
Instead of these three columns, why not have two columns: one for version and one for actively supported/EOL?